deploy/helm/kubecf/assets/operations/instance_groups/app-autoscaler.yaml

```diff
@@ -0,0 +1,1040 @@
+      common_name: metricsserver client
+      extended_key_usage:
+      - client_auth
+- type: replace
+  path: /variables/-
+  value:
+    name: postgres_ca
+    type: certificate
+    options:
+      common_name: postgresCA
+      is_ca: true
+- type: replace
+  path: /variables/-
+  value:
+    name: postgres_server
+    type: certificate
+    options:
+      ca: postgres_ca
+      common_name: autoscalerpostgres.service.cf.internal
+      extended_key_usage:
+      - client_auth
+      - server_auth
+
+# Add autoscaler bpm ops.
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=scalingengine/properties/quarks?
+  value:
+    ports:
+    - name: scalingengine
+      protocol: TCP
+      internal: 6104
+    - name: sehealthport
+      portocol: TCP
+      internal: 6204
+    bpm:
+      processes:
+      - name: scalingengine
+        executable: /bin/bash
+        args:
+        - -c
+        - |
+          /var/vcap/jobs/scalingengine/bin/scalingengine_ctl start
+    run:
+      healthcheck:
+        scalingengine:
+          readiness:
+            exec:
+              command:
+              - curl
+              - --fail
+              - --head
+              - http://127.0.0.1:6204
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=scheduler/properties/quarks?
+  value:
+    ports:
+    - name: scheduler
+      protocol: TCP
+      internal: 6102
+    - name: sdhealthport
+      portocol: TCP
+      internal: 6202
+    bpm:
+      processes:
+      - name: scheduler
+        executable: /bin/bash
+        args:
+        - -c
+        - |
+          /var/vcap/jobs/scheduler/bin/scheduler start
+    run:
+      healthcheck:
+        scheduler:
+          readiness:
+            exec:
+              command:
+              - curl
+              - --fail
+              - --head
+              - http://127.0.0.1:6202
+- type: replace
+  path: /instance_groups/name=asactors/jobs/name=operator/properties/quarks?
+  value:
+    ports:
+    - name: ophealthport
+      protocol: TCP
+      internal: 6208
+    bpm:
+      processes:
+      - name: operator
+        executable: /bin/bash
+        args:
+        - -c
+        - |
+          /var/vcap/jobs/operator/bin/operator_ctl start
+    run:
+      healthcheck:
+        operator:
+          readiness:
+            exec:
+              command:
+              - curl
+              - --fail
+              - --head
+              - http://127.0.0.1:6208
+- type: replace
+  path: /instance_groups/name=asmetrics/jobs/name=metricsserver/properties/quarks?
+  value:
+    ports:
+    - name: metrics
+      protocol: TCP
+      internal: 6103
+    - name: mscollector
+      protocol: TCP
+      internal: 7103
+    - name: mshealthport
+      portocol: TCP
+      internal: 6303
+    bpm:
+      processes:
+      - name: metricsserver
+        executable: /bin/bash
+        args:
+        - -c
+        - |
+          /var/vcap/jobs/metricsserver/bin/metricsserver_ctl start
```

Comment:
This seems odd. Why isn't it just `[/var/vcap/jobs/metricsserver/bin/metricsserver_ctl, start]`, or, if you really need to run under bash, `[/bin/bash, /var/vcap/jobs/metricsserver/bin/metricsserver_ctl, start]`?

Reply:
It does not support `[/var/vcap/jobs/metricsserver/bin/metricsserver_ctl, start]` and `/var/vcap/jobs/metricsserver/bin/metricsserver_ctl start`. It support other 2 ways:
1) 
```
bpm:
      processes:
      - name: metricsserver
        executable: /var/vcap/jobs/metricsserver/bin/metricsserver_ctl
        args:
        - start
```
2)
```
bpm:
      processes:
      - name: metricsgateway
        executable: /var/vcap/packages/metricsserver/metricsserver
        args:
        - -c
        - /var/vcap/jobs/metricsserver/config/metricsserver.yml
```